settings/site.ini

```diff
@@ -562,6 +562,7 @@ UserCreatorID=14
 # md5_user generates password hash from user and password.
 # md5_site generates password hash from site, user and password
 # plaintext does not generate a hash but has the password as it is, this is not
+# bcrypt generates password bcrypt from password only.
```

Comment:
Should be added on the line bellow, as the comment above continues to next line ( '... is not recommented ... ')